### ISSUE TITLE
Impossible to change Trigger Shortcut bug

### ISSUE THREAD

**Issue by Atantares** [iSapozhnik/Launchy-App-Switcher-Launcher#1]
Can't set custom shortcut to trigger menu. The app recognize only action keys (e.g. Command, Option, Shift...), so I can't set shortcut like Command + Shift + _any_key_here_

<img width="632" alt="image" src="https://github.com/user-attachments/assets/31b17dda-5cb1-47ef-a13b-5860f4966c3b" />

**Comment by iSapozhnik** [comment]
Hey Vladyslav!

The idea was to show the app only while certain keys (modifier keys) are being pressed. So, instead of hitting a combo to make the app appear, you actually need to keep the keys pressed. I assumed people wouldn’t want to hold down a combination of modifier keys and regular keys simultaneously. From that perspective, it works as designed. However, let’s keep the issue open and see how many people disagree with this approach.

**Comment by technolag** [comment]
Hey Ivan!

Jumping on to this one. It looks like the way it's designed currently does not allow BTT to integrate with customer actions to trigger the hotkeys. BTT only allows modifiers + regular keys.

I use trackpad gestures for everything. If I wanted to Four Finger Tap to trigger Launchy, I would set the hot key in BTT to be the same as Launchy for my Four Finger Tap. But I need an extra key to be able to do that. 

I don't like using keyboard shortcuts as I prefer to keep my hand on trackpad as much as possible as it's way faster for me. 

Would love the ability to at least set the shortcut to include non-modifier keys.

**Comment by maneeshpatel777** [comment]
yeah change this, I was like this is insane. I had a free key combo perfect for this app, option + tab, and I can't use it. also shame no trackpad way of doing it, if the shortcut was sorted, could use another tool to remap this.

please fix it, from Jun 2025!

**Comment by btthle** [comment]
New user here and having the same problem as everyone above. The limitation to only let modefier keys trigger launchy is hilarious. this makes it a pain in the ass to setup with other trackpad gestures. BTT needs it like this: modifier keys+any other key in order to be added as a trigger (modefier keys alone are not accepted). Any other workarounds with advanced scripting and tinkering can be easily avoided by giving users the option to not only use modefier keys but also combine them with any other generic key (optional).

Good and useful app, sadly not buying until this has been changed. @iSapozhnik

**Comment by eftalor** [comment]
@iSapozhnik my take on it:

You got me hooked to a level I think this thing should be a system default as it improves the flow of my work, and is intuitive, I think humans like round shapes better than a long rectangle that takes most of the screen

**Current:** 
Combinations of only modifier keys are available (ie ctrl, option, cmd) 

**Suggested:**
Have the ability to to change Launchy's _trigger shortcut_ to use not only the modifier keys but also additional keys: 

**Justification:**
Instead of invoking _option+cmd_ , would like to have an option to assign custom trigger shortcut, mainly:
_option+tab_
If there is a possibility to even replace the default MacOS one that would be cool (but I assume there is an OS limitation) 

either the _option+tab_ , or the _cmd+tab_ would be even more intuitive to the workflow and will make application usage smoother.

A different way I would suggest is having presets, this way you can limit the amount of trigger shortcuts and have only a few, for example: 

_option+cmd
ctrl+cmd
ctrl+option
option+tab_


Not neccesarily replace the current behavior, but having this as additional option would be awesome

Thank you for a great app! already a satisfied customer of Lasso :-)

**Comment by iSapozhnik** [comment]
It has been a while :) But I have some good news. I am testing the new build where the shortcuts logic was rewritten from scratch. Inviting everyone to check it out: https://discord.com/channels/1388471767835873351/1515442628626219018

**Comment by iSapozhnik** [comment]
@eftalor ;)

https://github.com/user-attachments/assets/9d4346d7-2eee-457f-ad20-1f5ef383268c
